src/Microsoft.VisualStudio.Services.Agent/JobNotification.cs

```diff
@@ -28,11 +28,11 @@ public sealed class JobNotification : AgentService, IJobNotification
         private bool _useSockets = false;
         private bool _isMonitorConfigured = false;
 
-        public async Task JobStarted(Guid jobId, string accessToken, Uri serverUrl)
+        public async Task JobStarted(Guid jobId, string accessToken, Uri serverUrl, Guid planId, string identifier, string definitionId, string planType)
```

Comment:
Interface signature doesn't match the implementation -> variable names and typos.